code/game/gamemodes/endgame/bluespace_jump/bluespace_jump.dm

```diff
@@ -72,70 +72,48 @@
 /obj/effect/bluegoast
 	name = "bluespace echo"
 	desc = "It's not going to punch you, is it?"
-	var/mob/living/carbon/human/daddy
+	var/mob/living/carbon/human/real_one
 	anchored = TRUE
 	var/reality = 0
 	simulated = FALSE
 
-/obj/effect/bluegoast/New(nloc, ndaddy)
+/obj/effect/bluegoast/New(nloc, nreal_one)
 	..(nloc)
-	if(!ndaddy)
+	if(!nreal_one)
 		qdel(src)
 		return
-	daddy = ndaddy
-	set_dir(daddy.dir)
-	appearance = daddy.appearance
-	GLOB.moved_event.register(daddy, src, /obj/effect/bluegoast/proc/mirror)
-	GLOB.dir_set_event.register(daddy, src, /obj/effect/bluegoast/proc/mirror_dir)
-	GLOB.destroyed_event.register(daddy, src, /datum/proc/qdel_self)
+	real_one = nreal_one
+	set_dir(real_one.dir)
+	appearance = real_one.appearance
+	GLOB.moved_event.register(real_one, src, /obj/effect/bluegoast/proc/mirror)
+	GLOB.dir_set_event.register(real_one, src, /obj/effect/bluegoast/proc/mirror_dir)
+	GLOB.destroyed_event.register(real_one, src, /datum/proc/qdel_self)
 
 /obj/effect/bluegoast/Destroy()
-	GLOB.destroyed_event.unregister(daddy, src)
-	GLOB.dir_set_event.unregister(daddy, src)
-	GLOB.moved_event.unregister(daddy, src)
-	daddy = null
+	GLOB.destroyed_event.unregister(real_one, src)
+	GLOB.dir_set_event.unregister(real_one, src)
+	GLOB.moved_event.unregister(real_one, src)
+	real_one = null
 	. = ..()
 
 /obj/effect/bluegoast/proc/mirror(var/atom/movable/am, var/old_loc, var/new_loc)
 	var/ndir = get_dir(new_loc,old_loc)
-	appearance = daddy.appearance
+	appearance = real_one.appearance
 	var/nloc = get_step(src, ndir)
 	if(nloc)
 		forceMove(nloc)
 	if(nloc == new_loc)
-		reality++
-		if(reality > 5)
-			to_chat(daddy, "<span class='notice'>Yep, it's certainly the other one. Your existance was a glitch, and it's finally being mended...</span>")
-			blueswitch()
-		else if(reality > 3)
-			to_chat(daddy, "<span class='danger'>Something is definitely wrong. Why do you think YOU are the original?</span>")
-		else
-			to_chat(daddy, "<span class='warning'>You feel a bit less real. Which one of you two was original again?..</span>")
+		to_chat(real_one, "<span class='warning'>You feel a bit less real. Which one of you two was original again?..</span>")
```

Comment:
While you're here, you should wrap this in a `SPAN_WARNING` instead of using html spans.

Reply:
I'd much rather replace all HTML spans in another PR.

Reply:
I'd prefer you fix this one while you're around to lessen the immediate work, but since you're the host, I can't really stop you.

Reply:
You are as much a maintainer as I am in those regards, no need to bring up my position as host.
I'll replace all HTML spans in this file with define, but other files will be edited in next PRs.